config/gitleaks.toml

```diff
@@ -14,17 +14,21 @@ title = "gitleaks config"
 description = "global allow lists"
 regexes = [
     '''(?i)^true|false|null$''',
+    '''^(?i:[a]+|[b]+|[c]+|[d]+|[e]+|[f]+|[g]+|[h]+|[i]+|[j]+|[k]+|[l]+|[m]+|[n]+|[o]+|[p]+|[q]+|[r]+|[s]+|[t]+|[u]+|[v]+|[w]+|[x]+|[y]+|[z]+|[*]+)$''',
```

Comment:
@rgmz have you found anything in the wild where this is necessary?

Reply:
I guess for example secrets right?

Reply:
Yeah. It's one of the most common example placeholders I've seen.

Albeit,  it's predominantly a/x/y/z/*.